### ISSUE TITLE
[Request] different hiddenDays for multiple weeks

### ISSUE THREAD

**Issue by arca223** [fullcalendar/fullcalendar#3130]
Hello,

It would be interesting to be able to give an array to hiddenDays, with the number of week (which you can get with Moment().week()) to be able to specify the days you'd like to hide in specific week.

hiddenDays: [
{
    week: 1,
    days: [0,1,2]
},
{
    week: 2,
    days: [3,4,5,6]
}]

That way, it'd allow people to disable outside range dates in correlation with disabling prev, next when outside date range. 

What do you think about ?

**Comment by arshaw** [comment]
i see you what you mean, but it would probably be more useful to specify specific dates to disabled, like:

```
hiddenDays: [
    '2016-01-01',
    '2016-01-02'
]
```

**Comment by arca223** [comment]
I agree, but this might change the code algorithm ? And if implemented this way, people using hiddenDays will see their applications broken on update.

What I tried to explain (and missed a part) is to be able to either give just the array of days to hide as the actual function, or give an array of week/days.
I don't know what changes should be done on the code, the easiest to do. And also not breaking applications.

**Comment by arshaw** [comment]
@arca223 you're right, we might want to introduce a new setting called `hiddenDates` to prevent breakage. also agree that a generator function might be a good idea

**Comment by arshaw** [comment]
very related to #1581, which deals with how do explicit define what days to render, rather than which days to hide

**Comment by KonnectSalon** [comment]
Any update on this?
